stdlib/warnings.pyi

```diff
@@ -1,9 +1,11 @@
 from types import ModuleType, TracebackType
-from typing import Any, List, TextIO, Type, overload
+from typing import Any, List, Optional, TextIO, Tuple, Type, overload
 from typing_extensions import Literal
 
 from _warnings import warn as warn, warn_explicit as warn_explicit
 
+filters: List[Tuple[str, Optional[str], Type[Warning], Optional[str], int]]
```

Comment:
- We now prefer the new `|` union syntax and lowercase `list` and `tuple`. I recently created #5872 and #5888 to change existing code. (Mypy is buggy. If `tuple` doesn't work, we can go back to `Tuple`.)
- I think the filters list shouldn't be exposed as mutable in `warnings.pyi`. It's fine to have it mutable in `_warnings.pyi` though: if you are brave enough to `import _warnings`, you probably don't want to be limited by this.

```suggestion
filters: Sequence[tuple[str, str | None, Type[Warning], str | None, int]]
```

You have to add `from typing import Sequence` and remove `Optional`.